refs/schemas/protocols/RFC-PRMP.md

```diff
@@ -0,0 +1,126 @@
+
+## Requirements
+
+- Systems exchange PRIV objects related to Data Subjects they both know of. The way they do it MUST be:
+    - Asynchronous: A System may be unavailable and get a message when it becomes available again
+    - Non-blocking: Systems may take action with regards to PRIV objects they know of without waiting for any input from other Systems
+    - [Eventually Consistent](https://en.wikipedia.org/wiki/Eventual_consistency) with regards to PRIV objects [a System is concerned by](#priv-objects-that-concern-a-system)
+
+- Systems SHOULD NOT have access to the content of PRIV objects not concerning them (e.g. Privacy Requests of users they do not know of, or Privacy Requests Responses from other Systems that may be confidential)
+
+- Systems CAN implement only parts of the Protocol. E.g. a browser may only send Privacy Requests, Consents and retrieve nothing (or retrieve only Privacy Request Responses). We call such systems **Agent Systems**. We call Systems that perform actual data processing and that implement the full Protocol - **Processing Systems**.
+
+- Systems are independent in their responses, i.e. a System MUST BE ABLE to respond to a Privacy Request addressed to it regardless of (the existence and the content of) the responses of other Systems.
+
+- The Systems CAN apply the Protocol in the context of different [Scenarios][Scenarios].
+
+- The Protocol can be used regardless of the Systems exchanging data in a one-off data dump, or through a continuous connection.
+
+- The network of Systems exchanging data related to a Data Subject CAN be cyclical. It MUST not be assumed to be a complete graph. It CAN also be highly volatile i.e. Systems can participate on an ad-hoc basis, only once, disappear and reappear.
+
+## Design Choices
+
+- The Protocol is agnostic from any particular way to address Systems and transfer messages to and from them. Systems are responsible of knowing how to identify and address another System with which they exchange data or any object defined in PRIV.
+
+- Systems process Privacy Requests and other PRIV objects (e.g. legal-base related events and consents) in order of their dates, and regardless of the order in which they have gain knowledge of them using the Protocol.
+
+- The Protocol is agnostic of the level of inter-system trust. A System MAY be implemented to trust other Systems to authenticate the Data Subject, covey responses, tell the truth or not. The security implications of such choices are out of scope of the Protocol.
+
+- The content of a message exchanged between two Systems is a PRIV object (e.g. [Privacy Request](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#privacy-request), [Privacy Request Response](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#privacy-request), [Consent](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#consent))
```

Comment:
When Consent is exchanged, what implications does it have to a consenting user?
Should the PCE compute the data to be deleted?

Reply:
> When Consent is exchanged, what implications does it have to a consenting user? 

The Eligible Privacy Scope is recalculated. So further processing may be constrained by this. Also some stored fragments (`processing-category:STORING`) might no longer be associated to an eligible privacy scope. 

> Should the PCE compute the data to be deleted?

Yes (as said above, if data gets out of Eligible Privacy Scope) then PCE should suggest deletion.

Reply:
And when should the consent be transferred?

Reply:
Whenever there is a new concent that concerns a System, it is transfered to that System.